packages/react/src/patterns/sections/CardSection/__stories__/CardSection.stories.js

```diff
@@ -2,11 +2,11 @@ import './index.scss';
 import { object, select, withKnobs } from '@storybook/addon-knobs';
 import cards from './data/cards.json';
 import CardSection from '../CardSection';
+import CardSectionSimple from '../CardSectionSimple';
 import { DDS_CARD_SECTION } from '../../../../internal/FeatureFlags';
```

Comment:
@mkothur need to remove `FeatureFlags`

Reply:
@kennylam: Okay, thought to remove but in the story did not mentioned so?

Reply:
@mkothur Yes looks like it was left out of the acceptance criteria, but all of these patterns should have FeatureFlags removed.